### DISCUSSION TITLE
22 marzo 2022 - esercizio 3

### DISCUSSION THREAD

**Discussion by IonutCicio** [sapienzastudentsnetwork/basi-di-dati-1#10] [Esami]
![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/40034429/7fc0aaf2-648d-405a-b762-a2f6a974fd7e)

**Comment by niccolo-fato** [comment]
![2024-01-13 10 06 15](https://github.com/sapienzastudentsnetwork/bdi2324/assets/73995334/04f33859-a874-42dc-9916-48b95337f696)
